src/ILCompiler.CppCodeGen/src/CppCodeGen/CppWriter.cs

```diff
@@ -1015,6 +1020,7 @@ private String GetCodeForObjectNode(ObjectNode node, NodeFactory factory, bool g
             List<NodeDataSection> nodeDataSections = new List<NodeDataSection>();
             byte[] actualData = new byte[nodeData.Data.Length];
             Relocation[] relocs = nodeData.Relocs;
+            long[] relocOffsets = new long[relocs.Length];
```

Comment:
This does not need to be `long`. It can be `int`. Relocs are never going to be more than 32-bit.

Reply:
Thank you!